app/src/main/java/org/systers/mentorship/view/fragments/EditProfileFragment.kt

```diff
@@ -55,7 +55,7 @@ class EditProfileFragment: DialogFragment() {
                 }
             }
         })
-        dialog.window!!.setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_ADJUST_RESIZE)
+        dialog?.window!!.setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_ADJUST_RESIZE)
```

Comment:
This is not a change that is related to Migrating to ViewPager 2. I think you'll have to remove this as well.

Reply:
it doesn't compile otherwise because of the updated support lib dependency